.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
+      }
+    ]
+  ],
+  "targets": {
```

Comment:
This also needs to be moved inside the preset config, next to `useBuiltIns`.

Reply:
Yeah, saw that as an issue [here](https://github.com/babel/babel/issues/6854). Forgot where I copied that from, but apparently, that was WRONG!